src/metaschema/oscal_control-common_metaschema.xml

```diff
@@ -205,6 +213,14 @@
         </define-assembly>
         <define-assembly name="parameter-guideline">
                 <formal-name>Guideline</formal-name>
+                <!-- 
+                        TODO: Is `@ns` flag enough or should we add system for
+                        dictating guidelines?
+
+                        Use cases: RMF recommends one thing, maybe different with
+                        FedRAMP, then a contradiction or subset of constraints is
+                        required, not recommended for example FCEB agency (Important Federal Agency).
+                -->
```

Comment:
```suggestion
```

Reply:
Sounds good to me, added https://github.com/david-waltermire-nist/OSCAL/issues/54 for follow up. Dave will convert this to a usnistgov/OSCAL issue after our review meeting. (Sorry, Dave.)